### ISSUE TITLE
1.1.1 < 1.0.7  Really?

### ISSUE THREAD

**Issue by russel** [sdkman/sdkman-cli#546]
I give you:
```
|> sdk outdated
==== BROADCAST =================================================================
* 15/03/17: Kotlin 1.0.7 released on SDKMAN! #kotlin
* 14/03/17: Kotlin 1.1.1 released on SDKMAN! #kotlin
* 14/03/17: Asciidoctorj 1.5.5 released on SDKMAN! #asciidoctorj
================================================================================

Upgrade:
kotlin (1.1.1 < 1.0.7)

Upgrade candidate(s) and set latest version(s) as default? (Y/n): n
```
and ask WTF.

**Comment by marc0der** [comment]
This is due to the Kotlin team releasing 1.0.7 after 1.1.1 and marking it as default. This is not the first time either...

**Comment by marc0der** [comment]
I've just corrected this for 1.1.1 once again. I've raised an [issue](https://youtrack.jetbrains.com/issue/KT-16871) on their issue tracker regarding this recurring problem.

**Comment by russel** [comment]
I have added my vote to the issue.
